.github/workflows/docs_deploy.yml

```diff
@@ -19,19 +19,30 @@ jobs:
         with:
           # We need to fetch the whole history so 'reno' can do its job and we can inspect tags.
           fetch-depth: 0
-
       - uses: actions/setup-python@v5
         name: Install Python
         with:
           # Sync with 'documentationPythonVersion' in 'azure-pipelines.yml'.
           python-version: '3.9'
-
       - name: Install dependencies
```

Comment:
Minor nit: please can we restore the spacing lines between the job stages?  There's another deletion below the `tox -e docs` line as well.